jack-and-the-beans/buddies-ios
# BUD-116 Topic tab and Pick Topic collection items restyle

Really simple stuff here.  
* Titles for Topic items are bigger
* Buttons are bigger than they appear (easier to tap)
* Checkboxes in Pick Topics view via Create Activity
* Renamed the `TopicCell` collection to `TopicImageCollection` to avoid confusion with @noahtallen 's work

_fyi @noahtallen I gave up on nibs because it just wasn't worth it for the really weird bug I ran into._

I'm putting this out here now, but **don't merge it until @JakeThurman 's search component is merged**.  They both change the same storyboard and I can easily redo mine to avoid a conflict.

To test:
* Verify the hitbox for the buttons are more user friendly
* See how you like the Topics tab
   * Verify favoriting a topic persists locally and to Firebase
* See how you like the Pick Topics window
   * Verify clicking properly adds to the Create Activity screen
